### PR TITLE
Added Crunchy link for SukaSuka

### PR DESCRIPTION
**Pull request by ad3z10**
Somebody in the comments pointed out that it was missing.

### CODE REVIEW

File: season_configs/spring_2017.yaml [hunk 1]
```diff
@@ -167,7 +167,7 @@ info:
   anilist: 'https://anilist.co/anime/21860/ShuumatsuNaniShitemasukaIsogashiidesukaSukutteMoratteIidesuka'
   mal: 'https://myanimelist.net/anime/33502/Shuumatsu_Nani_Shitemasu_ka_Isogashii_desu_ka_Sukutte_Moratte_Ii_desu_ka'
 streams:
-  crunchyroll: ''
+  crunchyroll: 'http://www.crunchyroll.com/worldend-what-do-you-do-at-the-end-of-the-world-are-you-busy-will-you-save-us'
```
Comment (janclarin) [inline season_configs/spring_2017.yaml:170]: Consider changing it to https because security :)

**Review by janclarin — COMMENTED** [review]
Https

File: season_configs/spring_2017.yaml (hunk 1 above)
Comment (ad3z10) [reply season_configs/spring_2017.yaml:170]: @janclarin that's actually on Crunchy not me, most of their site (including all videos) doesn't support https.

**Comment by mnmt** [comment]
Hey @ad3z10, although I may have missed the odd one, I have already made a pull request #27 which has all the newer CR links including the SukaSuka link. I think TheEnigmaBlade has been away for a bit, and hasn't been able to look over it yet.

**Comment by ad3z10** [comment]
@mnmt good to know, I'll take down my pull request for now, should have really checked but its been a while since I used GitHub so I got lost in the UI a little. 

If any other links you're missing come up before Enigmablade gets back I'll just tag you on your pull request then.

**Pull request closed without merging**